### PR TITLE
test(alerts): Speed up create alert rule test

### PR DESCRIPTION
**Pull request by scttcper**
Switch to paste event as userEvent.type in our forms has performance issues, shaves 2 seconds off this one test.

Can see the typing happening 3 times each taking most of the time of the entire test. It looks like we have already used paste a few times in this file for this reason.

image

- cleans up some `waitFor` that weren't needed
- removes a few things that weren't used anymore like the routes

### CODE REVIEW

**Review by JonasBa — APPROVED** [review]

**Comment by JonasBa** [comment]
Nice find, @scttcper. I've been looking at what's happening with this test in https://github.com/getsentry/sentry/pull/32075, so far not many actionable leads. We seem to spend much of the test suite duration just waiting for files :/

File: tests/js/spec/views/alerts/create.spec.jsx [hunk 1]
```diff
@@ -338,16 +253,14 @@ describe('ProjectAlertsCreate', function () {
...
       // Change target environment
       await selectEvent.select(screen.getByText('All Environments'), ['production']);
 
       // Change actionMatch and filterMatch dropdown
-      await selectEvent.select(screen.getAllByText('all')[0], ['any']);
-      await selectEvent.select(screen.getAllByText('all')[0], ['any']);
+      const allDropdowns = screen.getAllByText('all');
+      expect(allDropdowns).toHaveLength(2);
+      await selectEvent.select(allDropdowns[0], ['any']);
+      await selectEvent.select(allDropdowns[1], ['any']);
```
Comment (priscilawebdev) [inline tests/js/spec/views/alerts/create.spec.jsx:263]: do we need multiple await here?

**Review by priscilawebdev — APPROVED** [review]
left 1 question but it looks good to me. Unfortunately `userEvent.type` slows down the test :/

File: tests/js/spec/views/alerts/create.spec.jsx (hunk 1 above)
Comment (scttcper) [reply tests/js/spec/views/alerts/create.spec.jsx:263]: we could promise.all them 🤷 but yes its two separate dropdown actions

**Comment by scttcper** [comment]
@JonasBa oh i hadn't seen your pr looks like we tried some of the same things. The files at least get cached i think so they just happen to get run through babel on the first test that has them in the dependency tree. It might be interesting to see if we can fix how deep our dependency tree goes. Most the files loaded aren't used in this test.

**Pull request merged**